packages/react/src/date-range-picker/utils.test.ts

```diff
@@ -96,37 +91,3 @@ describe('getCalendarDefaultMonth', () => {
 		).toEqual(1990);
 	});
 });
-
-describe('getFromDateInputButtonAriaLabel', () => {
```

Comment:
Removed these for one generic util for use in both date pickers. Will also help us when we optimise this button label generation